1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,9 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy = store
+.filter(product => product.price < money)
+.map(product => product.item)
```

Comment:
good use of array methods.
What happens if the `money` is exactly the price of an item?